docs/docs/creating-global-styles.md

```diff
@@ -2,7 +2,36 @@
 title: Creating global styles
 ---
 
-This is a stub. Help our community expand it.
+In this guide, we'll walk through using traditional global styles in your site.
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+First, open a new terminal window and run the following commands to create a new test site and start the development server:
+
+```shell
+gatsby new global-style-tutorial https://github.com/gatsbyjs/gatsby-starter-hello-world#v2
+gatsby develop
+```
+
+Second, create a css file and define any styles you wish. An arbitrary example:
+
+```css
+html {
+    background-color: lavenderblush;
+}
+
+a {
+    color: rebeccapurple;
+}
+```
+
+Then, include the stylesheet in your site's `gatsby-browser.js` file.
+
+```javascript
+import './src/styles/global.css'
+
+// or:
+// require('./src/styles/global.css')
+```
+
+> _Note: You can use Node.js require or import syntax._
+
+You should see your global styles taking effect across your site.
```

Comment:
I know this is a pain, but could you add a screenshot to show the output?

Reply:
Not a pain -- I was going off the `/docs/styled-components` page. can add.

Reply:
Added. Also changed this to use default-starter instead of hello-world, since it already has a `gatsby-browser.js` file, makes the example slightly simpler.